README.md

```diff
@@ -118,38 +112,24 @@ You should see the following:
 
 #### Python 3
 
-To install Python 3 on Windows is pretty straightforward.
+Installing Python 3 on Windows is pretty straightforward.
 
 Visit the download page for [Python 3](https://python.org/downloads). You should see the link for Python 3.10.4:
 
 ![win-python](docs/images/win-python.png)
 
-#### Add DUNE to Path
-
-To keep from having to install files to the user's system, the preferred method of usage is to add this directory to your `Path`. 
-Go to Start Search and type `edit the system` and choose `Edit the system environment variables`.
-
-<img src="docs/images/win-edit-sys.png" alt="win-edit-sys" width="200">
-
-From here you should click on the button marked `Environment Variables`.
-
-<img src="docs/images/win-sys-props.png" alt="win-sys-props" width="250">
-
-From here select the line with `Path` and select `Edit...`
-
-<img src="docs/images/env-vars-1.png" alt="env-vars-1" width="300">
-
-From here add a new line with the location of the downloaded DUNE project:
+Directory containing python.exe should be added to environment variable PATH.
+Open cmd.exe and verify `python --version` returns current Python version.
 
-<img src="docs/images/env-vars-2.png" alt="env-vars-2" width="300">
+#### DUNE installation on Windows <a name="dune-windows"></a>
 
-### Bootstrap DUNE
-
-Pick your preferred command line application and input the following command:
-
-```console:
-C:\<PathToDUNE>\DUNE$ .\bootstrap.bat
-```
+1. Install [Chocolatey](https://docs.chocolatey.org/en-us/choco/setup)
+2. From the [latest release](https://github.com/AntelopeIO/DUNE/releases/latest) page, download latest DUNE nupkg or visit the [release tags](https://github.com/AntelopeIO/DUNE/releases) page to download specific version of DUNE.
+3. Run Windows Console as an Administrator, go to the directory where `nupkg` file has been downloaded and run:
+`choco install antelopeio-dune.1.0.0.nupkg`
```

Comment:
> What if run:
> `choco install https://github.com/AntelopeIO/DUNE/packaging/antelopeio-dune.nuspec`
> Would it works?

No, to my knowledge it's not possible to install directly from URL.

> `curl https://github.com/AntelopeIO/DUNE/packaging/antelopeio-dune.nuspec`
> and then:
> `choco install antelopeio-dune.nuspec`
> curl.exe exists in Windows from 10

That would be possible, but I wouldn't spend time on this workaround because https://github.com/AntelopeIO/DUNE/issues/95 would allow to simply do:
`choco install antelopeio-dune`